documentation/configuration/collection-rule-configuration.md

```diff
@@ -598,6 +599,60 @@ Usage that collects a full dump and egresses it to a provider named "AzureBlobDu
   ```
 </details>
 
+### `CollectExceptions` Action
```

Comment:
Not something we _need_ to do (and we could always do it later), but could be good to incorporate this into one of our [Collection Rule Examples](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/collectionrules/collectionruleexamples.md)

Reply:
I updated the existing example that triggers on 5 HTTP 4xx requests to collect exceptions instead of a dump; there's another similar example below this that collects a dump and opens it in VS. I think it's more natural to collect exceptions for the ASP.NET response failure example.